FAQ.rst

```diff
@@ -0,0 +1,18 @@
+FAQ
+===
+
+Here are some answers to frequently-asked questions from IRC and elsewhere.
+Got a question that isn't answered here? Try `IRC <irc://irc.freenode.net/fedora-apps>`_,
+the `mailing list <https://lists.fedorahosted.org/mailman/listinfo/packagedb>`_.
+
+How do I...
+===========
+
+...specify comaintainers on a new package request?
+--------------------------------------------------
+
+Once the package is created you can add other packagers, pseudo-users or groups:
+
+    1. Go to one of your package: https://admin.fedoraproject.org/pkgdb/package/apache-commons-chain/
```

Comment:
Maybe change the package name to something else ? For example ``<package name>`` ?